server/src/models/merchants/index.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview This file contains the database operations related to Merchant
+ * entities.
+ * @author Karen Frilya Celine
+ */
+
+import {Datastore} from '@google-cloud/datastore';
+import {Guid} from 'guid-typescript';
+
+const datastore = new Datastore();
+const KIND = 'Merchant';
+
+async function getMerchants() {
```

Comment:
It's not clear what data is returned. Should we create a return type or add comments to the method?

Reply:
Do you mean that it's not clear what a merchant object is like? I think adding a return type array of merchants would be good but we need to create a merchant class first.
I think we can create another PR for that.

Reply:
Do you think it's better to `getAllMerchants` instead? This way we can potentially avoid typo oopsies

Reply:
@hzxcaryn good point! but does it make sense if we have filters for the merchants?

Reply:
Alternatively you could call it 'ListMerchants'. All SGTM

Reply:
okay I changed to `getAllMerchants`